modules/remotecontrol/constants.js

```diff
@@ -1,31 +1,28 @@
 module.exports = {
     /**
-     * Types of remote-control-event events.
+     * Types of remote-control events.
      */
-    EVENT_TYPES: {
+    EVENTS: {
         mousemove: "mousemove",
         mousedown: "mousedown",
         mouseup: "mouseup",
         mousedblclick: "mousedblclick",
         mousescroll: "mousescroll",
         keydown: "keydown",
         keyup: "keyup",
-        permissions: "permissions",
         stop: "stop",
         supported: "supported"
     },
 
     /**
-     * Actions for the remote control permission events.
+     * Types of remote-control requests.
```

Comment:
So should all instances of "remote control" be hyphenated?  I see comments below, old and new, that are not hyphenated.